Ginger/Ginger/ApplicationModelsLib/POMModels/POMWizardLib/POMObjectsMappingWizardPage.xaml.cs

```diff
@@ -118,20 +136,17 @@ private async void Learn()
             if (!mWizard.IsLearningWasDone)
             {
                 mWizard.ProcessStarted();
-                pomAllElementsPage.unmappedUIElementsPage.DriverIsBusy = true;
+                mWizard.Agent.Driver.IsDriverBusy = true;
                 xStopLoadButton.Visibility = Visibility.Visible;
```

Comment:
you should not set the Driver.IsDriverBusy value from outside the Driver, set only get{} to IsDriverBusy and add private mIsDriverBusy  which will be set inside the Driver based on it Activities- only Driver set for it self if it Busy or not, from outside use Driver.IsDriverBusy only as indiication to know if Driver is free to get new calls or not